### ISSUE TITLE
How to run as server and connect as client from different machine?

### ISSUE THREAD

**Issue by KrishnaPG** [alibaba/GraphScope#4393]
The documentation asks to install graphscope as python package. It is not clear if graphscope is an embedded database or can be run as standalone server.

For example, how to launch graphscope as a server running on remote machine and connect to it from a different machine as client?

Also, from the code there seems to be some .proto files in it - can they be used to generate a javascript grpc client and use that to connect to the server instance on remote machine?

In other words, how can I connect from a Javascript client to Graphscope running on a different machine?

Any help would be greatly appreciated.

**Comment by KrishnaPG** [comment]
Any answer for this please? Neither on the Slack nor here any answer
